canonical/lxd-ui
# Add tests for cluster groups WD-4882

## Done

- added playwright tests for cluster group CRUD

Fixes WD-4882

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - check the tests in the PR are successful
